examples/acm-terraform-blog-part3/README.md

```diff
@@ -0,0 +1,88 @@
+# Enable ACM features with Terraform - Part 3
+
+This is Part3 of the tutorial to accompany a short series of  blog articles explaining how to enable [Anthos Config Management (ACM)](https://cloud.google.com/anthos/config-management) with Terraform.
+
+In the [first part](../acm-terraform-blog-part1), we explained how to use Teraform to create a cluster and manage its config from git via [Config Sync](https://cloud.google.com/anthos-config-management/docs/config-sync-overview).
+
+In the [second part](../acm-terraform-blog-part2) we added guard rails for the cluster configuration via [Policy Controller](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller).
+
+In this article we'll demonstrate how, using Config Connector, you can provision your GCP cloud resources following the same GitOps K8s native model.
+
+## Provision GCP resources
+
+1. Set the variable for the project from [part2](../acm-terraform-blog-part2). We will re-use that project but create a new cluster since we cleaned up at the end of the first section. If you are working in a different project, enable required GCP APIs, as described in [part1/README.md](../part1/README.md).
```

Comment:
```suggestion
1. Set the variable for the project from [part two](../acm-terraform-blog-part2). We will re-use that project but create a new cluster since we cleaned up at the end of the first section. If you are working in a different project, enable required GCP APIs, as described in [part one](../part1/README.md).
```